src/preload.ts

```diff
@@ -229,6 +235,30 @@ const electronAPI = {
   showContextMenu: (options?: ElectronContextMenuOptions): void => {
     return ipcRenderer.send(IPC_CHANNELS.SHOW_CONTEXT_MENU, options);
   },
+  Config: {
+    /**
+     * Controls whether ComfyUI core will start up with the `--cpu` flag.
+     * @param settings An key/value pair object containing all settings to update
+     */
+    setCpuMode: async (enabled: boolean): Promise<void> => {
```

Comment:
Where is this `setCpuMode` function get called? I cannot find its usage in the frontend PR: https://github.com/Comfy-Org/ComfyUI_frontend/pull/1888/files